spec/element_spec.rb

```diff
@@ -1,135 +1,140 @@
 require 'watirspec_helper'
 
-describe Watir::Element do
+module Watir
```

Comment:
In classes we should prefer:
```
module Foo
  class Bar
...
```
to 
```
class Foo::Bar
...
```
This change was along those lines, and also means we don't have to specify the Watir namespace everywhere in the specs.

That being said, I didn't make sure this was consistent in other files, so I should probably remove it for now.